pan3d/ui/pan3d_components/module/__init__.py

```diff
@@ -1,7 +1,7 @@
 from pathlib import Path
 
 # Compute local path to serve
-serve_path = str(Path(__file__).parent.with_name("serve").resolve())
+serve_path = str(Path('pan3d/ui/pan3d_components/module'))
```

Comment:
__Old__ `str(Path(__file__).parent.with_name("serve").resolve())`
__New__ `str(Path(__file__).with_name("serve").resolve())`

Reply:
The module directory should contains all it needs.

Reply:
Do you mean that I should move the `serve` directory within the `module` directory? Otherwise this new line results in a "no directory exists" error.

Reply:
yes, you need to fix both for consistency.

Reply:
Ok, changed in [06bc8bd](https://github.com/Kitware/geo-trame/pull/78/commits/06bc8bd0840bba4f662dbd1d16ac0d9180d69190)